examples/jsm/animation/CCDIKSolver.js

```diff
@@ -217,7 +217,8 @@ class CCDIKSolver {
 	 */
 	createHelper() {
 
-		return new CCDIKHelper( this.mesh, this.mesh.geometry.userData.MMD.iks );
+		const iks = this.mesh.geometry.userData.MMD?.iks ?? this.iks;
+		return new CCDIKHelper( this.mesh, iks );
```

Comment:
If I'm right, `CCDIKSolver` no longer needs to have a dependency with `mesh.geometry.userData.MMD` so just passing `this.iks` would be good enough.

```suggestion
		return new CCDIKHelper( this.mesh, this.iks );
```